src/com/google/javascript/jscomp/parsing/parser/Scanner.java

```diff
@@ -651,15 +651,32 @@ private Token scanIdentifierOrKeyword(int beginToken, char ch) {
     valueBuilder.append(ch);
 
     boolean containsUnicodeEscape = ch == '\\';
```

Comment:
Actually, `containsUnicodeEscape` is unused now, so I guess you can just inline it?